sdv/multi_table/base.py

```diff
@@ -345,6 +345,10 @@ def sample(self, scale=1.0):
                 If ``scale`` is lower than ``1.0`` create fewer rows by the factor of ``scale``
                 than the original tables. Defaults to ``1.0``.
         """
+        if not isinstance(scale, (float, int)) or type(scale) is bool:
+            raise SynthesizerInputError('scale parameter can only be float or a int instance.')
+        elif scale < 0 or np.isnan(scale):
+            raise ValueError('scale parameter must be >0.0.')
```

Comment:
Should this also be a `SynthesizerInputError` instead of a `ValueError`?

Reply:
Made the change in the last commit